lib/logzio-nodejs.js

```diff
@@ -198,22 +199,33 @@ class LogzioLogger {
 
         if (this.addTimestampWithNanoSecs) {
             const time = process.hrtime();
-            msg['@timestamp_nano'] = msg['@timestamp_nano'] || [now, time[1].toString().padStart(nanoSecDigits ,'0')].join('-');
+            msg['@timestamp_nano'] = msg['@timestamp_nano'] || [now, time[1].toString().padStart(nanoSecDigits, '0')].join('-');
         }
     }
 
     log(msg, obj) {
         if (this.closed === true) {
             throw new Error('Logging into a logger that has been closed!');
         }
-        if (![null, undefined].includes(obj)){
+        if (![null, undefined].includes(obj)) {
             msg += JSON.stringify(obj);
         }
         if (typeof msg === 'string') {
             msg = {
                 message: msg,
             };
         }
+        const { en0 } = networkInterfaces();
```

Comment:
Please create it as a new function
Maybe host data and we will add the hostname as well?